server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -215,9 +219,16 @@ public List<Short> check(Environment env, Mutation mutation) {
         } catch (NumberFormatException | ArrayIndexOutOfBoundsException nfe) {
           violations = addViolation(violations, 1);
         }
-      } else if (columnFamily.equals(ScanFileColumnFamily.NAME)) {} else if (columnFamily
-          .equals(OnDemandAssignmentStateColumnFamily.NAME)) {} else if (columnFamily
-              .equals(BulkFileColumnFamily.NAME)) {
+      } else if (columnFamily.equals(ScanFileColumnFamily.NAME)) {
+
+      } else if (columnFamily.equals(HostingColumnFamily.GOAL_COLUMN.getColumnFamily())
+          && columnQualifier.equals(HostingColumnFamily.GOAL_COLUMN.getColumnQualifier())) {
```

Comment:
This can be shortened

```suggestion
      } else if (HostingColumnFamily.GOAL_COLUMN.equals(columnFamily,columnQualifier)) {
```

Reply:
Suggestion applied in cd223b6